extensions/wikia/ImageReview/ImageReviewSpecialController.class.php

```diff
@@ -116,32 +152,84 @@ public function stats() {
 		$endMonth = $this->request->getVal( 'endMonth', date( 'm' ) );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "endMonth" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)